dist/reacttutorial.min.js

```diff
@@ -3,4 +3,4 @@
e",Ae),n.removeEventListener("mouseup",He),Re()):qe(r,t),t.mouseDownReceived=!1}function Be(e){var t=c[ie];qe(t.selector?document.querySelector(t.selector):null,t),t.mouseDownReceived=!1}function Ue(e,t){var n=h(),r=n.w,o=n.h,i=p(ue.current),a=i.width,c=i.height;Oe(re(re({type:"HAS_DOM_NODE"},e),{},{helperWidth:a,helperHeight:c,helperPosition:t,w:r,h:o,inDOM:!0}))}function Ge(e){var t,n,r;(e.stopPropagation(),!0!==S)&&(S&&(t=S.includes("esc"),n=S.includes("right"),r=S.includes("left")),27!==e.keyCode||t||(e.preventDefault(),Ne()),39!==e.keyCode||n||(e.preventDefault(),Ee()),37!==e.keyCode||r||(e.preventDefault(),Ce()))}var Ke=c[ie]&&("function"==typeof c[ie].content?c[ie].content({close:Ne,goTo:Se,inDOM:xe.inDOM,step:ie+1}):c[ie].content);return u?o.a.createElement("div",{className:"react-tutorial"},o.a.createElement(W,{ref:ue,windowWidth:xe.w,windowHeight:xe.h,targetWidth:xe.width,targetHeight:xe.height,targetTop:xe.top,targetLeft:xe.left,targetRight:xe.right,targetBottom:xe.bottom,helperWidth:xe.helperWidth,helperHeight:xe.helperHeight,helperPosition:xe.helperPosition,arrowPosition:c[ie].arrowPosition,rounded:P,accentColor:N,defaultStyles:!0,padding:j,className:s()(X.base,O,oe({},X.isOpen,u))},x,Ke,k&&o.a.createElement(A,{"data-tour-elem":"badge"},ie+1),(b||g)&&o.a.createElement(B,{"data-tour-elem":"controls"},b&&o.a.createElement(q,{onClick:Ce,disabled:0===ie,label:v||null}),g&&o.a.createElement(G,{"data-tour-elem":"navigation"},c.map((function(e,t){return o.a.createElement(V,{key:"".concat(e.selector?e.selector:"undef","_").concat(t),onClick:function(){return Se(t)},current:ie,index:t,disabled:ie===t||M,showNumber:w,"data-tour-elem":"dot",className:s()(Z.base,oe({},Z.active,ie===t))})}))),b&&o.a.createElement(q,{onClick:ie===c.length-1?z?Ne:function(){}:Ee,disabled:!z&&ie===c.length-1,inverted:!0,label:z&&ie===c.length-1?z:H||null}))),o.a.createElement(F,{windowWidth:xe.w,windowHeight:xe.h,targetWidth:xe.width,targetHeight:xe.height,targetTop:xe.top,targetLeft:xe.left,padding:j,rounded:P,className:R,disableInteraction:!1===c[ie].stepInteraction||T?!c[ie].stepInteraction:T,maskColor:K,disableInteractionClassName:s()(Q.disableInteraction,C)})):null},se={top:0,right:0,bottom:0,left:0,width:0,height:0,w:0,h:0,node:null};function de(e,t){switch(t.type){case"HAS_DOM_NODE":t.type;var n=te(t,ee);return re(re({},e),n);case"NO_DOM_NODE":return re(re({},e),{},{top:e.h+10,right:e.w/2+9,bottom:e.h/2+9,left:t.w/2-e.helperWidth?e.helperWidth/2:0,width:0,height:0,w:t.w,h:t.h,helperPosition:"center",node:null});default:return e}}var fe={CLICK:"click",DBL_CLICK:"dblclick",TYPING:"typing",ENTER:"enter",DRAG_N_DROP:"dragdrop",DRAG_WITH_MOUSE_MOVE:"dragwithmove",CUSTOM:"custom",WAIT:"wait"};ue.propTypes=$,ue.defaultProps={showNavigation:!0,showNavigationNumber:!0,showButtons:!0,showNumber:!0,startAt:0,scrollDuration:1,maskSpace:10,stepWaitTimer:0,disableInteraction:!1,rounded:0,accentColor:"#007aff",closeWithMask:!0,maskColor:"#000"};var pe=ue;t.default=pe}])}));
```

Comment:
Let 's not commit minified files with changes
We always have separate commit of bundled min files with version update